modules/Bio/EnsEMBL/Hive/RunnableDB/FastaFactory.pm

```diff
@@ -148,6 +152,7 @@ sub write_output {
 
     my $input_seqio         = $self->param('input_seqio');
     my $max_chunk_length    = $self->param('max_chunk_length');
+    my $max_chunk_size      = $self->param('max_chunk_size');
```

Comment:
Need to define a default value in `param_defaults`

Reply:
Done.